### PR TITLE
[SCAL-206789] Add option to set default accordion state of defined custom groups

### PR DESCRIPTION
**Pull request by bishalts2019**

### CODE REVIEW

File: src/embed/app.ts [hunk 1]
```diff
@@ -374,6 +375,27 @@ export interface AppViewConfig extends Omit<ViewConfig, 'visibleTabs'> {
...
+     * @example
+     * ```js
+     * const embed = new AppEmbed('#embed', {
+     *   ... // other app view config
+     *   dataPanelCustomGroupsAccordionInitialState:
+     *      DataPanelCustomColumnGroupsAccordionState.EXPAND_ALL,
+     * });
+     * ```
+     */
+    dataPanelCustomGroupsAccordionInitialState?: string;
```
Comment (AdityaMittal31) [inline src/embed/app.ts:398]: instead of using string can we use the enum that we have created?

Comment (adityamittal3107) [inline src/embed/app.ts:398]: Can we use the enum created instead of using string?

File: src/index.ts [hunk 2]
```diff
@@ -48,6 +48,7 @@ import {
     LogLevel,
     VizPoint,
     CustomActionPayload,
+    DataPanelCustomColumnGroupsAccordionState,
```
Comment (adityamittal3107) [inline src/index.ts:51]: Why are we passing this in import/export?

Comment (bishalts2019) [reply src/index.ts:51]: I was following as Actions enum, if not required here let me remove it.

**Review by AdityaMittal31 — APPROVED** [review]

**Review by adityamittal3107 — APPROVED** [review]

**Pull request merged**